contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,356 @@
+and durable storage.
+
+Ideally release notes should [tell a story][] which
+is compelling enough to encourage users and operators to upgrade their clusters.
+Without a standardized mechanism for describing important enhancements our
+talented technical writers and product managers struggle to weave a coherent
+narrative explaining why a particular release is important. Additionally for
+critical infrastructure such as Kubernetes adopters need a forward looking road map
+in order to plan their adoption strategy.
+
+[tell a story]: https://blog.rust-lang.org/2017/08/31/Rust-1.20.html 
+[road to Go 2]: https://blog.golang.org/toward-go2
+[survey data]: http://opensourcesurvey.org/2017/
+
+## Detailed design
+
+### What type of work should be tracked by a KEP
+
+The definition of what constitutes an "enhancement" is a foundational concern
+for the Kubernetes project. Roughly any Kubernetes user or operator facing
+enhancement should follow the KEP process: if an enhancement would be described
+in either written or verbal communication to anyone besides the KEP author or
+developer then consider creating a KEP. One concrete example is an enhancement
+which should be communicated to SIG Release or SIG PM.
+
+Without detailed information explaining the motivation for an enhancement SIGs
+must first approve a proposal, agreeing to a motivation over a mailing list,
+video call, or hallway conversation. During the release process this motivation
+must be rediscovered by the SIG, hopefully by finding a design proposal. The
+process of announcing an enhancement through release notes suggests another
+heuristic for describing what work should be tracked through an KEP: anything
+that would require a design proposal. In fact it is possible to consider a KEP
+an enhancement to the design proposal process in which design proposals are
+used throughout the process of proposing an enhancement, scoping its design,
+tracking its implementation, and agreeing on criteria for graduation to general
+availability.
+
+As the local bodies of governance SIGs should have broad latitude in describing
+what constitutes an enhancement which should be tracked through the KEP process;
+hopefully the desire to be included in the rich narrative of a release announcement
+will encourage broad participation in the KEP process across SIGs. SIGs should
+also have the freedom to customize the KEP template according to their SIG
+specific concerns. For example the KEP template used to track API changes will
+likely have different subsections than the template for proposing governance
+changes.
+
+### KEP Template
+
+The template for a KEP will be submitted in a separate proposal
+
+### KEP Workflow
+
+A KEP is proposed to have the following states
+
+- **opened**: a new KEP has been filed but not triaged by the responsible SIG or
+  working group
+- **accepted**: the motivation has been accepted by the SIG or working group as in
```

Comment:
Is this approved? PTAL at the metadata proposal. We should make the 2 consistent.

Reply:
Yes, I think `accepted` is the same as `approved` but whatever the precise term used I was trying to suggest that when a proposal was merged it means that the responsible SIGs have agreed on at least the motivation being in roadmap

Reply:
Diverging from existing terminology on the project would be confusing.